nd4j/nd4j-backends/nd4j-backend-impls/nd4j-native/pom.xml

```diff
@@ -204,17 +204,6 @@
                         <version>${openblas.version}-${javacpp-presets.version}</version>
                         <classifier>${dependency.platform}</classifier>
                     </dependency>
-                    <dependency>
-                        <groupId>org.bytedeco</groupId>
-                        <artifactId>mkl-dnn</artifactId>
-                        <version>${mkl-dnn.javacpp.version}</version>
-                    </dependency>
-                    <dependency>
-                        <groupId>org.bytedeco</groupId>
-                        <artifactId>mkl-dnn</artifactId>
-                        <version>${mkl-dnn.javacpp.version}</version>
-                        <classifier>${dependency.platform2}</classifier>
-                    </dependency>
```

Comment:
Like I mentioned previously, replace those with the full version of MKL. There is usually no reason to prefer OpenBLAS over MKL!

Reply:
so, something like this?

```
<artifactId>mkl</artifactId>
<version>${mkl.javacpp.version}</version>
<classifier>${dependency.platform2}</classifier>
```

Reply:
Yes, something like that. Current version is `2019.4-1.5.1`. Not too sure if we need all the same dependencies, but just replacing everything mechanically for now should give something that works.

Reply:
Ok, I'll open new PR with MKL tomorrow morning, and get back to you.